test/pages/business/UtrYesNoPageSpec.scala

```diff
@@ -33,15 +33,15 @@ class UtrYesNoPageSpec extends PageBehaviours {
     beRemovable[Boolean](UtrYesNoPage)
 
     "implement cleanup logic when NO selected" in {
-      val userAnswers = UserAnswers("id", "utr", "sessionId", LocalDate.now)
+      val userAnswers = UserAnswers("id", "utr", "sessionId", "newId", LocalDate.now)
```

Comment:
this is wrong, maybe it is working in tests but it is not doing the same logic as in repository, you are suppose to use combination of id and utr and sessionId

Reply:
done